core/trino-main/pom.xml

```diff
@@ -371,7 +371,7 @@
         <dependency>
             <groupId>net.java.dev.jna</groupId>
             <artifactId>jna-platform</artifactId>
-            <version>5.12.1</version>
+            <version>${dep.jna.version}</version>
```

Comment:
Sorry, just noticed another issue. We can remove the version here, since it's already set in dependency management. Thus, no need to add a property for it, just update the version directly in dependency management.

Reply:
It's not the same dependency `jna` vs `jna-platform`

Reply:
I've moved the `jna-platform` to the `dependencyManagement` in the parent pom